docs/analysis.md

```diff
@@ -243,11 +243,11 @@ trial.collections()
 ```
 
 ###### Returns
-`dict[str -> Collection]` A dictionary indexed by the name of the collection, with the Collection object as the value. Please refer [Collection API](api.md) for more details. #TODO fix link
+`dict[str -> Collection]` A dictionary indexed by the name of the collection, with the Collection object as the value. Please refer [Collection API](https://github.com/awslabs/sagemaker-debugger/blob/master/docs/api.md) for more details.
```

Comment:
Could you point to this https://github.com/awslabs/sagemaker-debugger/blob/master/docs/api.md#collection

Reply:
Done.